woocommerce/woocommerce-gateway-stripe
# Update workflow runners from ubuntu-18.04 to ubuntu-latest

Fixes #2590 

## Changes proposed in this Pull Request:

- Updates all workflows using `ubuntu-18.04` to `ubuntu-latest`.

## Testing instructions

Check that the workflows are working in this PR.

---


**Post merge**
